packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -155,15 +157,62 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
         // TODO: Mark msg.sender as already voted on electionVotes
     }
 
-    function _isNextEpochNomination() internal view virtual returns (bool) {
-        return false;
+    function epochFinised() external view override returns (bool) {
+        if (_electionStore().epochStart == 0) {
+            revert EpochNotStarted();
+        }
+
+        return _epochFinished();
+    }
+
+    function isEpochNominationPeriod() external view override returns (bool) {
+        if (_electionStore().epochStart == 0) {
+            revert EpochNotStarted();
+        }
+
+        return _isEpochNominationPeriod();
+    }
+
+    function isEpochVotingPeriod() external view override returns (bool) {
+        if (_electionStore().epochStart == 0) {
+            revert EpochNotStarted();
+        }
+
+        return _isEpochVotingPeriod();
+    }
+
+    function initialize() external override {
+        // TODO set epoch 0 seat to 1
+        // TODO set epoch 0 period
+        // TODO set token for owner
+        _changeEpoch();
```

Comment:
OZ syndrome?

Reply:
originally (and still not sure) I thought the changeEpoch can be called from more than one place... but yes, up to now is only once, so agree it looks OZed